ci/docker_integration_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -i
+
+if [ -z "$branch_specifier" ]; then
+    # manual
+    IMAGE_NAME="logstash-integration-tests"
+else
+    # Jenkins
+    IMAGE_NAME=$branch_specifier"-"$(date +%s%N)
+fi
+
+echo "Running Docker CI build for '$IMAGE_NAME' "
+
+docker build  -t $IMAGE_NAME .
+exit_code=$?; [[ $exit_code != 0 ]] && exit $exit_code
+docker run -t --rm $IMAGE_NAME ci/integration_tests.sh $@
+exit_code=$?
+[[ $IMAGE_NAME != "logstash-integration-tests" ]] && docker rmi $IMAGE_NAME
```

Comment:
@jakelandis Nit: you might want to `trap` the image delete if you want to be nice to whoever maintains Jenkins :)

Reply:
then again ... nah let's not do that, it gets tricky :) Kind of unlikely to actually have build the image and still getting exit `!=0`from the build not sure if it's even possible.